_posts/2016-07-28-ticket-sales-start-monday-august-1st.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "Ticket Sales Start Monday August 1st - Save the Date!"
+socialImageSrc: '/assets/posts/tickets.png'
+---
+
+The news you've all been waiting for: you can finally secure your seats at RustFest in September.
+
+
+Starting Monday August 1st from 3pm (CEST) you can purchase supporter Tickets and View-Source-Combo-Tickets through our website. Exactly one week later you can buy the first batch of regular tickets. Thus giving you the chance to still buy one of the limited early-bird-combo-tickets before the deadline August 12th for a discount price.
+
+![](/assets/posts/tickets.png)
+
+
+Regular tickets will be **125€** (148,75€ incl. VAT), Supporter Tickets are priced at 250€. Until August 12th you can get the View Source Combo Ticket for 350€, which will be 450€ after that. A student discount of 20% will be available on Regular Tickets.
+
+We will distribute Regular Tickets in batches, opening at varying times of day giving an appropriate notice beforehand. So, if don't get any this time, another batch will come. Further more we will be distributing tickets through [diversitytickets.org](http://diversitytickets.org).
```

Comment:
- `Regular Tickets` -> `regular tickets`
- So, if __you__ don't get any this time?
- `Further more` -> `Furthermore`